src/distributions/uvnormal.jl

```diff
@@ -0,0 +1,15 @@
+# add docstr
+function Normal(μ::Number, Σ::Number)
+    T = promote_type(typeof(μ), typeof(Σ))
+    return Normal{T}(convert(T, μ), convert(T, Σ))
+end
+
+const UvNormal{T} = Normal{T,T,T} where {T<:Number}
+#const UvNormal{T,V} = Union{Normal{T,T,T},Normal{T,T,V}} where {T<:Real,V<:Complex{T}}
+
+cov(N::UvNormal) = real(N.Σ)
```

Comment:
Why does this have to be tranformed into real?

Reply:
It no longer does. Good catch, I decided to simply store the covariance as real anyway.